### PR TITLE
[MIG] account_invoice_line_report: Migration to 10.0

### PR DESCRIPTION
**Pull request by binhlam**
Migrate module `account_invoice_line_report` from v9.0 to v10.0.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please squash together adjacent "OCA Transbot.." commits using https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by binhlam** [comment]
@pedrobaeza  OK, I've just squashed the commits.

File: account_invoice_line_report/report/account_invoice_report_view.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <field name="res_model">account.invoice.report</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,pivot,graph</field>
+        <field name="search_view_id" ref="view_account_invoice_report_search"/>
+        <field name="help">From this report, you can have an overview of the amount invoiced to your customers. The search tool can also be used to personalise your Invoices reports and so, match this analysis to your needs.</field>
+    </record>
+
+    <menuitem action="action_account_invoice_line_report" id="menu_action_account_invoice_line_report" parent="account.account_reports_business_intelligence_menu" sequence="25"/>
+
+</odoo>
```
Comment (jcdrubay) [inline account_invoice_line_report/report/account_invoice_report_view.xml]: Missing last line

File: account_invoice_line_report/README.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+it first, help us smashing it by providing a detailed and welcomed feedback.
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
```
Comment (jcdrubay) [inline account_invoice_line_report/README.rst:43]: You should add the original author:

    Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>

File: account_invoice_line_report/report/account_invoice_report_view.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Binh Lam - Komit Consulting <binh.lt@komit-consulting.com>
```
Comment (jcdrubay) [inline account_invoice_line_report/report/account_invoice_report_view.xml]: You should add the original copyright:

<!-- Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>

File: account_invoice_line_report/README.rst (hunk 2 above)
Comment (jcdrubay) [reply account_invoice_line_report/README.rst:43]: You should also add yourself

File: account_invoice_line_report/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Binh Lam <binh.lt@komit-consulting.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Account Invoice Line Report',
+    'summary': 'New views to manage invoice lines information',
+    'version': '10.0.1.0.0',
+    'category': 'Account',
+    'author': 'Binh Lam, '
```
Comment (jcdrubay) [inline account_invoice_line_report/__manifest__.py]: You should add the original author: `Tecnativa`

you should add yourself aslo.

**Review by jcdrubay — CHANGES_REQUESTED** [review]

**Comment by binhlam** [comment]
Hi, I updated the original author.

**Review by jcdrubay — APPROVED** [review]

**Review by congdpt — APPROVED** [review]

**Comment by pedrobaeza** [comment]
You have made latest changes amending the "OCA Transbot..." commit, so you have mixed things. Please start again without mixing commits.

**Comment by binhlam** [comment]
@pedrobaeza Hi, I've just committed again. Please check, thanks.

File: account_invoice_line_report/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Account Invoice Line Report',
+    'summary': 'New views to manage invoice lines information',
+    'version': '10.0.1.0.0',
+    'category': 'Account',
+    'author': 'Tecnativa, '
+              'Binh Lam (komit-consulting.com), '
```
Comment (pedrobaeza) [inline account_invoice_line_report/__manifest__.py]: Please add only companies, not person

File: account_invoice_line_report/report/account_invoice_report_view.xml [hunk 6]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Copyright 2017 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+     Copyright 2018 Komit <http://komit-consulting.com>
```
Comment (pedrobaeza) [inline account_invoice_line_report/report/account_invoice_report_view.xml]: You are not really modifying this file (only removing a commented line), so the copyright addition doesn't apply

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_invoice_line_report/report/account_invoice_report_view.xml (hunk 6 above)
Comment (binhlam) [reply account_invoice_line_report/report/account_invoice_report_view.xml]: I changed those.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**